charts/s3gw/templates/certificate.yaml

```diff
@@ -4,35 +4,35 @@
 apiVersion: cert-manager.io/v1
 kind: Certificate
 metadata:
-  name: s3gw-ca-cert
+  name: {{ .Release.Name }}-{{ .Release.Namespace }}-ca-cert
```

Comment:
using ` {{ include "s3gw.serviceName" . }}` instead of  `{{ .Release.Name }}-{{ .Release.Namespace }}` does not ensure the uniqueness between installations.
Does this make sense?